internal/api/grpc/settings/v2/settings.go

```diff
@@ -131,6 +136,39 @@ func (s *Server) GetActiveIdentityProviders(ctx context.Context, req *settings.G
 	}, nil
 }
 
+func activeIdentityProvidersToQuery(req *settings.GetActiveIdentityProvidersRequest) (_ []query.SearchQuery, err error) {
+	q := make([]query.SearchQuery, 0)
+	if req.GetCreationAllowed() {
+		creationQuery, err := query.NewIDPTemplateIsCreationAllowedSearchQuery(true)
```

Comment:
this prevents filtering for IdPs, which do not allow creation (also check the following checks)

Reply:
Added booleans as false as well, including tests